Framework/Algorithms/src/MergeRuns/SampleLogsBehaviour.cpp

```diff
@@ -371,6 +376,22 @@ void SampleLogsBehaviour::mergeSampleLogs(MatrixWorkspace &addeeWS,
 }
 
 /**
+ * Perform the update for a sum property, adding a new value to the existing
+ *one. Skipped if the time series log entry is in the addeeWS.
+ *
+ * @param addeeWSNumber the sample log value from the workspace being merged
+ * @param outWSNumber the sample log value for the merged workspace
+ * @param outWS the workspace the others are merged into
+ * @param name the name of the property
+ */
+void SampleLogsBehaviour::updateSumProperty(double addeeWSNumber,
+                                            double outWSNumber,
```

Comment:
I wanted to convey it is a number not, say, a string value. I have gone for `addeeWSNumericValue`.